From e40eeee2f1e9311b1581a5bbf42adb28fd816c51 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 1 Apr 2025 23:16:29 +0000
Subject: [PATCH] window: Switch to empty view on failure

Without this, the window stucks at loading when invalid project is passed.
---
 src/widgets/window.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/widgets/window.rs b/src/widgets/window.rs
index b0504ad..566f958 100644
--- a/src/widgets/window.rs
+++ b/src/widgets/window.rs
@@ -243,6 +243,7 @@ impl Window {
             self.set_open_project(project);
         } else {
             tracing::error!("Failed to parse the project");
+            self.set_view(View::Empty);
         }
     }
 
-- 
GitLab

